### ISSUE TITLE
Split file structure for PyTorch YOLO recasts

### ISSUE THREAD

**Issue by harryjmoss** [UCL-RITS/vetii#7]
[YOLOv5](https://github.com/ultralytics/yolov5/) (and I think v3, but we should use v5) etc requires consecutively numbered files in training and validation splits, otherwise it thinks files are missing.

Need to split the yolo images and labels off from the main test/validation set and rename them, making sure that the label file still corresponds to the right image file.
